handlers/serviceset.go

```diff
@@ -167,6 +176,22 @@ func (s *ServiceSet) logError(format string, args ...interface{}) {
 	}
 }
 
+// reloadEndpointHandler
+func (s *ServiceSet) reloadEndpointHandler(w http.ResponseWriter, r *http.Request) {
+	token, err := r.URL.Query().Get("token")
+	if s.enableReloadEndpoint && token == s.reloadToken {
+		_, err := w.Write([]byte("OK"))
+		if err != nil {
+			http.NotFound(w, r)
```

Comment:
I have literally no idea about go, but this looks like we're swallowing an error, which is often wrong.

Reply:
I did this as a bit of obfuscation on the server, if a token isn't provided correctly then don't hit that the endpoint exists. Happy to change it though if that seems silly, thoughts?

Reply:
oh i see you're referencing an error with the write, that makes sense